server/src/main/java/org/opensearch/throttling/admissioncontroller/settings/IOBasedAdmissionControllerSettings.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.throttling.admissioncontroller.settings;
+
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.throttling.admissioncontroller.controllers.IOBasedAdmissionController;
+import org.opensearch.throttling.admissioncontroller.enums.AdmissionControlMode;
+import org.opensearch.throttling.admissioncontroller.enums.TransportRequestActionType;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Settings related to io based admission controller.
+ * @opensearch.internal
+ */
+public class IOBasedAdmissionControllerSettings {
+
+    /**
+     * Default parameters for the IOBasedAdmissionControllerSettings
+     */
+    public static class Defaults {
+        public static final String MODE = "disabled";
+        public static List<String> TRANSPORT_LAYER_DEFAULT_URI_TYPE = Arrays.asList("indexing", "search");
```

Comment:
Can we complete this flow - mapping the actions to the actual transport actual names or whichever way we're planning to use to intercept the actions ?